app/assets/stylesheets/custom/course-menu.less

```diff
@@ -0,0 +1,8 @@
+/*******************************
+           .course-menu
+*******************************/
+
+.ui.course-menu.menu,
+.ui.course-menu.item {
+  border: 0 !important;
```

Comment:
I copied this stylesheet from `main-menu.less`.

Reply:
The reason `.main-menu` has all of these chained classes is because I was trying to override default `semantic-ui` styles without using `!important` (whether or not that was a good idea is indeed debatable 😅 ). That's not necessary here, since you *are* using `!important` and don't need the added specificity of the chained classes.